osm2gtfs/creators/managua/managua.json

```diff
@@ -0,0 +1,34 @@
+{
+    "query": {
+        "bbox": {
+            "n": "12.21",
+            "s": "12.04",
+            "e": "-86.12",
+            "w": "-86.38"
+        },
+        "tags": {
+            "route":"bus",
+            "network": "IRTRAMMA",
+            "public_transport:version": "2"
+        }
+    },
+    "agency": {
+        "agency_id": "NI-IRTRAMMA",
+        "agency_name": "Instituto Regulador del Transporte del Municipio de Managua",
+        "agency_url": "http://www.managua.gob.ni/",
+        "agency_timezone": "America/Managua",
+        "agency_lang": "es",
+        "agency_phone": "+505 255-2189",
+        "agency_fare_url": ""
+    },
+    "feed_info": {
+        "publisher_name": "MapaNica.net",
+        "publisher_url": "https://mapanica.net",
+        "version":  "0.1",
+        "start_date": "20171101",
+        "end_date": "20181031"
```

Comment:
`version`, `start_date` and `end_date` should be set by timetable.json, but can stay here too.

Reply:
I'm not sure about this - for me they are more important in the config file, than in the data source. See [comment here](https://github.com/grote/osm2gtfs/issues/62#issuecomment-339120679) and following. The state of the discussion was to allow both.

Reply:
Okay, perfect. Thanks for the explanation! Leave it here then 😄